### ISSUE TITLE
feat: include "replying to" in copy text

### ISSUE THREAD

**Issue by insertish** [revoltchat/frontend#172]

**Comment by sharkaccino** [comment]
if nobody else does, i might try tackling this after #173 / #182

**Comment by kimlimjustin** [comment]
@JackDotJS have you started working on this? I've figured out a way to handle this, what left is just to open a PR

**Comment by sharkaccino** [comment]
nope, i'm still waiting for approval on the other PR x) feel free to take this one!

**Comment by sharkaccino** [comment]
seeing as #188 kinda fell apart a while back, i figured i'd try sorting this out myself x)
